openmc/cell.py

```diff
@@ -0,0 +1,448 @@
+from collections import OrderedDict, Iterable
+from numbers import Real, Integral
+from xml.etree import ElementTree as ET
+import sys
+import warnings
+
+import openmc
+import openmc.checkvalue as cv
+from openmc.surface import Halfspace
+from openmc.region import Region, Intersection, Complement
+
+if sys.version_info[0] >= 3:
+    basestring = str
+
```

Comment:
Seems like there are a few extra newlines here and below `reset_auto_cell_id()`.